resources/migrations/000_migrations.yaml

```diff
@@ -11636,6 +11636,240 @@ databaseChangeLog:
+                  type: ${timestamp_type}
+                  defaultValueComputed: current_timestamp
+                  constraints:
+                    nullable: false
+              - column:
+                  remarks: The timestamp of when the timeline was updated
+                  name: updated_at
+                  type: ${timestamp_type}
+                  defaultValueComputed: current_timestamp
+                  constraints:
+                    nullable: false
+              - column:
+                  remarks: Type of action
+                  name: type
+                  type: ${text.type}
+                  constraints:
+                    nullable: false
+  - changeSet:
+      id: v44.00-013
+      author: snoe
+      comment: Added 0.44.0 - writeback
+      changes:
+        - createTable:
+            tableName: query_action
+            remarks: A readwrite query type of action
+            columns:
+              - column:
+                  name: action_id
+                  type: int
+                  remarks: The related action
+                  constraints:
+                    nullable: false
+                    referencedTableName: action
+                    referencedColumnNames: id
+                    foreignKeyName: fk_query_action_ref_action_id
+                    deleteCascade: true
+              - column:
+                  name: card_id
+                  type: int
+                  remarks: The related card
+                  constraints:
+                    nullable: false
+                    referencedTableName: report_card
+                    referencedColumnNames: id
+                    foreignKeyName: fk_query_action_ref_card_id
+                    deleteCascade: true
+  - changeSet:
+      id: v44.00-014
+      author: snoe
+      comment: Added 0.44.0 - writeback
+      changes:
+        - addPrimaryKey:
+            tableName: query_action
+            columnNames: action_id, card_id
+            constraintName: pk_query_action
+  - changeSet:
+      id: v44.00-015
+      author: snoe
+      comment: Added 0.44.0 - writeback
+      changes:
+        - createTable:
+            tableName: emitter
+            remarks: An Emitter reperents the fact an Action can act upon the given object
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  remarks: The timestamp of when the timeline was created
+                  name: created_at
+                  type: ${timestamp_type}
+                  defaultValueComputed: current_timestamp
+                  constraints:
+                    nullable: false
+              - column:
+                  remarks: JSON with vis settings and other things frontend related
+                  name: viz_settings
```

Comment:
@salsakran mentioned this https://github.com/metabase/eng-tech-designs/pull/7#discussion_r874894748 but something more generic like `options` might make more sense because we might be putting other stuff in this column in the future